dbt/adapters/trino/relation.py

```diff
@@ -13,3 +14,12 @@ class TrinoQuotePolicy(Policy):
 @dataclass(frozen=True, eq=False, repr=False)
 class TrinoRelation(BaseRelation):
     quote_policy: TrinoQuotePolicy = TrinoQuotePolicy()
+
+    # customized matches method from BaseRelation but ignore case
+    # because currently trino metastore is case-intensitive
+    def _is_exactish_match(self, field: ComponentName, value: str) -> bool:
+        # if self.dbt_created and self.quote_policy.get_part(field) is False:
```

Comment:
Please remove the the existing implementation details in the overridden method.

Reply:
done by https://github.com/bachng2017/dbt-trino/commit/8b5336b2b2d26391dc209460968087d2a785af31